### ISSUE TITLE
hk.BatchNorm with jax.vmap

### ISSUE THREAD

**Issue by reemabdelrazek30** [google-deepmind/dm-haiku#786]
Is there any workaround that I can perform in order to use jax.vmap with hk.BatchNorm. should I use hk.vmap instead? should I write a custom batchNorm?
